blas/gonum/level3double.go

```diff
@@ -463,17 +463,31 @@ func (Implementation) Dsyrk(ul blas.Uplo, tA blas.Transpose, n, k int, alpha flo
 			for i := 0; i < n; i++ {
 				ctmp := c[i*ldc+i : i*ldc+n]
 				atmp := a[i*lda : i*lda+k]
-				for jc, vc := range ctmp {
```

Comment:
How did this present?

Reply:
As I comment below, the bug was not here.